.github/workflows/cross-version-tests.yml

```diff
@@ -64,8 +64,10 @@ jobs:
     outputs:
       matrix1: ${{ steps.set-matrix.outputs.matrix1 }}
       matrix2: ${{ steps.set-matrix.outputs.matrix2 }}
+      matrix3: ${{ steps.set-matrix.outputs.matrix2 }}
```

Comment:
We're reaching the limit (matrix1 and matrix2 have about 200 items, the maximum is 256). Added a new group.